.github/workflows/forge-gcp.yaml

```diff
@@ -60,13 +60,19 @@ jobs:
 
       - uses: actions/setup-python@13ae5bb136fac2878aff31522b9efb785519f984 # pin@v4
 
-      - name: Install python deps
```

Comment:
This is not backwards compatible.... they both need to exist or be cherry picked in

Reply:
Do you mean cherry-picked in the release branches ?